ontotext-yasgui-web-component/src/models/plugins/pivot-table/pivot-table-renderer-name.ts

```diff
@@ -0,0 +1,14 @@
+export enum PivotTableRendererName {
```

Comment:
should these be translated

Reply:
These names are hardcoded into the pivot table libraries. I will examine the code, and if I find a way to translate them, I will do it.